tests/foldr/pass_frenchris.js

```diff
@@ -0,0 +1,7 @@
+const foldr = (fn, acc, el) => {
+  el = el.reverse()
```

Comment:
`reverse` mutate the array and so should not work.
we should have a test that ensure that the array as not be modified, we should have this test for `map`, `reduce` and `filter` too

Also, I feel like the order of the arguments should be different: `(el, fn, acc)` to match javascript reduce more closely